publish.html

```diff
@@ -0,0 +1,580 @@
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h3>OMERO.web</h3>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/publishThumbnailWeb.jpg" class="align-center" src="images/publishThumbnailWeb.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<li><p class="first">Select <span class="bold">Make Movie...</span> from the drop-down menu to create a movie from an image.</p>
+
+<p>Select the format from the <span class="bold">Format</span> drop-down.</p>
+
+<p>Use the <span class="bold">Frame Rate</span> drop-down to select the frame rate.</p>
+
+<p>Set the desired time interval if appropriate.</p>
+
+<p>Set the desired Z-section interval if appropriate.</p>
+
+<h3>OMERO.insight</h3>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/publishMovieInsight.jpg" class="align-left" src="images/publishMovieInsight.jpg" style="width: 75%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h3>OMERO.web</h3>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/publishMovieWeb.jpg" class="align-left" src="images/publishMovieWeb.jpg" style="width: 80%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+</ol>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<h2><a class="anchor" id="public"></a>Using a public group</h2>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<p>OMERO.web has the facility to allow public access to selected data without the need to log in. This functionality allows images to be made publicly accessible for embedding in web pages or use in other publishing modalities.</p>
+
+<p>A group is made publicly accessible by the addition of a Public User to the group by the OMERO system Admininistrator. Any group can therefore be made public, but it is usual to create groups specifically and give them the desired level of permissions, which is usually read-only.</p>
```

Comment:
random capitalisation of 'system Administrator' - you don't need the capital A, especially as you don't use one in the intro or the following examples.